### DISCUSSION TITLE
AI deck GAP8 bootloader issue.

### DISCUSSION THREAD

**Discussion by MAHASHANA** [bitcraze/discussions#1505] [Q&A]
Hi team,
i have a AI deck 1.1, I am stuck with the GPA 8 bootloader thing and i am unable to get my hands on the extra peripherals i.e. JTAG connector and  [ARM-USB-TINY-H](https://www.olimex.com/Products/ARM/JTAG/ARM-USB-TINY-H/). I would like to request you for a possible replacement or if possible any other method to resolve this issue. [FOI for me flashing firmware via the radio stops at 99% and does not finish all the time.]

**Comment by gemenerik** [comment]
Thank you for reaching out. I understand the difficulties you're experiencing with the GAP8 bootloader and not having the necessary peripherals. Note that both the product and store pages clearly state that a compatible JTAG debugger is required for setting up the AI deck. This is crucial for initial setup, updates, troubleshooting. There is no way to flash the bootloader without a debugger. Even if you receive a unit with the bootloader correctly installed, having a JTAG debugger is required, e.g. for bootloader updates. 

If obtaining these peripherals is not an option, please email support@bitcraze.io to inquire about a possible return.
